### DISCUSSION TITLE
Is it possible to implement an A4 page view, so svelte-lexical can act like google docs?

### DISCUSSION THREAD

**Discussion by mike-hogan** [umaranis/svelte-lexical#18] [Q&A] [answered]
Hello, title says it all really.  Seems like svelte-lexical is a continuous flow rich text editor.  Would it be possible to make it operate on a page basis?

**Comment by umaranis** [comment, marked as answer]
Hi @mike-hogan ,
It is NOT something currently possible with svelte-lexical out-of-the-box. 
It is definitely possible to implement such a feature but will require significant effort I guess.

PS: If you have any interest in working on development of such a feature, do let me know. Thanks.
